nvdaHelper/espeak/sconscript

```diff
@@ -138,6 +138,7 @@ espeakLib=env.SharedLibrary(
 		"phonemelist.c",
 		"readclause.c",
 		"setlengths.c",
+		"soundIcon.c",

```

Comment:
how did you know to add this here?

Reply:
The `updating-espeak.md` should cover.. or at least hint at this. The source espeak project is built from `Makefile.am`, I diffed this file (last commit we used vs new commit).

Reply:
I could clarify that in the `md` file?

Reply:
Ah right so it is only if the build fails? 

I was expecting this in the "doing the update"

Reply:
> Ah right so it is only if the build fails?

Good point, I guess the answer is "it depends". The build may not always fail, we explicitly exclude certain modules of espeak that we don't use. However this is only possible because of the way that these modules are implemented.  I'd say it is a good idea to always do that diff and make a decision about whether changes need to be made.